pkg/query-service/app/http_handler.go

```diff
@@ -5093,6 +5103,53 @@ func (aH *APIHandler) getDomainInfo(w http.ResponseWriter, r *http.Request) {
 	aH.Respond(w, finalResult)
 }
 
+// getSpanPercentileDetails handles requests for span percentile analysis
+func (aH *APIHandler) getSpanPercentileDetails(w http.ResponseWriter, r *http.Request) {
```

Comment:
since we have already created module, let's move the handler to the module itself.